src/pages/nfts/[address].tsx

```diff
@@ -205,37 +225,67 @@ export default function NftByAddress({
     address?: string;
     title: string;
     viewOnSite?: boolean;
-  }) => (
-    <div className={`flex items-center justify-between`}>
-      <p className={`m-0 text-base font-normal text-gray-300`}>{title}</p>
-      <div className={`flex flex-row items-center justify-end gap-2`}>
-        {viewOnSite && (
-          <Link href={`/collections/${address}`}>
-            <a target={`_self`}>
-              <FeatherIcon
-                icon="folder"
-                aria-hidden="true"
-                className={`h-4 w-4 text-white hover:text-gray-300`}
-              />
+  }) => {
+    const [linkCopied, setLinkCopied] = useState(false);
+
+    useEffect(() => {
+      if (linkCopied) {
+        const timer = setTimeout(() => {
+          setLinkCopied(false);
+        }, 3000);
+        return () => clearTimeout(timer);
+      }
+    }, [linkCopied]);
+
+    const handleCopyClick = async () => {
+      await navigator.clipboard.writeText(address || `Error`);
+      setLinkCopied(true);
+    };
+    return (
+      <div className={`flex items-center justify-between`}>
```

Comment:
change template literals to plain strings

Reply:
good catch an awk habit to break